core/otplease/otplease.js

```diff
@@ -66,7 +88,18 @@ function attempt(fn, opts, otpCache) {
           semaphore.release();
           return attempt(fn, opts.concat({ otp: otpCache.otp }), otpCache);
         }
-        return getOneTimePassword()
+        let basePromise = Promise.resolve();
+        let promptForPassword = () => getOneTimePassword();
+        if (rateLimitedOtp) {
+          basePromise = new Promise(resolve => {
+            const delay = opts.otpRateLimitDelay;
```

Comment:
Since we know it's rate limited, couldn't we prompt for a timeout period, perhaps providing a default value that would work fine in most cases? I'm concerned that folks may not always know that they might be rate-limited, as it might vary depending on the changeset. An explicitly-configured `--otp-rate-limit-delay` (or `command.publish.otpRateLimitDelay` in lerna.json) would skip this prompt and just start the timeout.

Reply:
- [ ] Add prompt for otp-rate-limite-delay if option is not passed *and* the error reported is related to rate limitations (with a default value being 5 minutes ?)

Reply:
While implementing the prompt I stumbled upon a tricky usability open question.

Since all of this may be in concurrent cases, I've already been facing the case where

* a first "thread" faces an OTP rate limit related error, starts locking and says he'll be waiting
* other threads continue their work until their reach the same stage (emitting all corresponding logs)
* the first thread which was stuck "waiting" now prompts for the otp, once it's done it releases its lock and everybody resumes

Implementing the prompt would mean that we'd have 

* a user being asked for some time value while other threads' logging would be stuck (due to the pausing)
* once the user enters a value all other threads' logging would resume (which could result in a *lot* of logs ;) ) and its wait would start (obviously I'm guessing the time reference would be before the lock is obtained, right ?)

Would that seem right ? (This is really an open question)